orc8r/cloud/go/obsidian/access/operator.go

```diff
@@ -25,59 +26,47 @@ import (
 	"github.com/labstack/echo"
 )
 
-// RequestOperator returns Identity of request's Operator (client)
+// getOperator returns Identity of request's Operator (client).
 // If either the request is missing TLS certificate headers or the certificate's
 // SN is not found by Certifier or one of certificate & its identity checks fail
 // - nil will be returned & the corresponding error logged
-func RequestOperator(c echo.Context) (*protos.Identity, error) {
-	if c == nil {
-		glog.Error("Nil Echo Context")
-		return nil, fmt.Errorf("Internal Server Error (Context)") // nil CTX, no useful info to log here
-	}
-	req := c.Request()
-	if req == nil {
-		glog.Error("Nil HTTP Request")
-		return nil, fmt.Errorf("Internal Server Error (Request)")
-	}
-
+func getOperator(c echo.Context, req *http.Request) (*protos.Identity, error) {
```

Comment:
This function has redundant parameter list, if you have Context, you should have http.Request